### PR TITLE
Added ports to default configuration.

### LINKED ISSUE #6132: Add Redis to default list of Opaque ports
Redis is pretty common. We should add it to the list of default skip ports.

https://github.com/linkerd/linkerd2/blob/da588130544154fdb94afb64147fa745ce852fc4/charts/linkerd2/values.yaml#L110

### PR DESCRIPTION
**Pull request by JasonMorgan**
Default Linkerd skip and opaque port configuration

Missing default ports based on [docs](https://linkerd.io/2.10/features/protocol-detection/)

Addressed: Add Redis to default list of Opaque ports #6132

Once merged, the default install values will match the recommendations in Linkerd's [TCP ports guide](https://linkerd.io/2.10/features/protocol-detection/).

Fixes #6132

### CODE REVIEW

File: charts/linkerd2/values.yaml [hunk 1]
```diff
@@ -105,16 +105,21 @@ proxy:
...
-  # -- Default set of inbound ports to skip via itpables
-  ignoreInboundPorts: ""
-  # -- Default set of outbound ports to skip via itpables
-  ignoreOutboundPorts: ""
+  # -- Default set of inbound ports to skip via iptables
+  # - Galera (4567,4568) UDP
+  ignoreInboundPorts: "4567,4568"
+  # -- Default set of outbound ports to skip via iptables
+  # - Galera (4567,4568) UDP
+  ignoreOutboundPorts: "4567,4568"
```
Comment (alpeb) [inline charts/linkerd2/values.yaml:122]: Note that UDP traffic is skipped by default by the proxy. According to Galera's docs, traffic on 4567 is TCP. I didn't see anything about 4568's protocol:
https://galeracluster.com/library/documentation/firewall-settings.html
https://galeracluster.com/library/documentation/glossary.html#term-cluster-replication
https://galeracluster.com/library/documentation/glossary.html#term-incremental-state-transfer
You can link to #5913 in these comments to keep a reference to the discussion about this case.

**Review by alpeb — COMMENTED** [review]
Thanks @JasonMorgan !

Could you also please followup with an update to the docs, so these ports also get the `none*` treatment? (no setup needed if using standard ports) :wink:

File: charts/linkerd2/values.yaml (hunk 1 above)
Comment (JasonMorgan) [reply charts/linkerd2/values.yaml:122]: Oh sorry, they just said TCP/UDP so I figured that's why we skipped them. I'll just remove the comment about why.

**Review by mateiidavid — COMMENTED** [review]
Hey @JasonMorgan this looks good! Think we need to have a look at the integration tests though and update them with the new ports.

Currently, the tests seem to fail because of something else though, there's an additional `--outbound-ports-to-ignore` field that doesn't seem to be expected:
```
exp: Args:[]string{"--incoming-proxy-port", "4143", "--outgoing-proxy-port", "4140", "--proxy-uid", "2102", "--inbound-ports-to-ignore", "4190,4191,1234,5678"}
act: {Args:[]string{"--incoming-proxy-port", "4143", "--outgoing-proxy-port", "4140", "--proxy-uid", "2102", "--inbound-ports-to-ignore", "4190,4191,1234,5678", "--outbound-ports-to-ignore", "4567,4568"}
diff: "--outbound-ports-to-ignore", "4567,4568"}
```


Additionally, here's a list of tests that _might_ have to be changed with the new default opaque ports:
```
cli/cmd/testdata/inject_emojivoto_deployment_debug.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_config_overrides.golden.yml
cli/cmd/testdata/inject_emojivoto_statefulset.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_empty_resources.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_controller_name.golden.yml
cli/cmd/testdata/inject_emojivoto_list.golden.yml
cli/cmd/testdata/inject_contour.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_udp.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment.golden.yml
cli/cmd/testdata/inject_emojivoto_pod_with_requests.golden.yml
cli/cmd/testdata/inject_tap_deployment_debug.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_no_init_container.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_opaque_ports.golden.yml
cli/cmd/testdata/inject_emojivoto_pod_proxyignores.golden.yml
cli/cmd/testdata/inject_gettest_deployment.good.golden.yml
cli/cmd/testdata/inject_emojivoto_pod.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_proxyignores.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_hostNetwork_false.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_overridden.golden.yml
cli/cmd/testdata/inject-filepath/expected/injected_redis.yaml
cli/cmd/testdata/inject-filepath/expected/injected_nginx.yaml
cli/cmd/testdata/inject-filepath/expected/injected_nginx_redis.yaml
cli/cmd/testdata/inject_emojivoto_list_empty_resources.golden.yml
cli/cmd/testdata/inject_emojivoto_pod_ingress.golden.yml
cli/cmd/testdata/inject_emojivoto_deployment_capabilities.golden.yml
cli/cmd/testdata/inject_emojivoto_already_injected.golden.yml
```

**Comment by dadjeibaah** [comment]
@JasonMorgan, I don't think I have permissions to push changes to this branch but I've included a diff that should fix the tests to get this to pass CI
https://gist.github.com/dadjeibaah/7162bb51f7ca0047a77a7d7d9d539135

you should be able to run something like:
`git apply pr.diff`

**Comment by JasonMorgan** [comment]
Thanks folks! @dadjeibaah I'll add your changes tonight. @alpeb I'll try and stage a docs PR at the same time.

**Review by alpeb — APPROVED** [review]
Thanks @JasonMorgan , this looks good to me, modulo merging back `main` into this branch and getting the conflicts fixed :+1: 

I'm gonna leave here references to #5913 and linkerd/website#1057 which have the discussions that led to the inclusion of the Galera ports in the list of opaque/ignoreInbound/ignoreOutbound lists.

**Comment by alpeb** [comment]
@JasonMorgan do you need any assistance to bring this one across the finish line? :slightly_smiling_face:

**Comment by JasonMorgan** [comment]
and conflicts should be deconflicted :) Sorry about that delay!

**Comment by JasonMorgan** [comment]
Just going to start this from scratch.

**Pull request closed without merging**